support-frontend/assets/pages/contributions-landing/components/ContributionTypeTabs.tsx

```diff
@@ -133,44 +134,6 @@ function ContributionTypeTabs(props: PropTypes) {
 		</ChoiceCardGroup>
 	);
 
-	// leaving in place as this is still in active development:
-	const renderControl = () => (
```

Comment:
This was introduced in a PR [2 yrs ago](https://github.com/guardian/support-frontend/pull/2418/commits/535ac04b49aa3e3da711996e7721f60abcde1a1d#diff-0ee75d3afdb02e24aa3b889d19e5462e2cb52b8fb5e9d80c6c7b36b11e71c823) and seems to have been unused since then